phonopy/interface/qe.py

```diff
@@ -315,7 +330,9 @@ def _set_lattice(self):
 
     def _set_positions(self):
         unit = self._values[0].lower()
-        if "crystal" not in unit:
+        if "angstrom" in unit:
+            self.scaled_positions = False
```

Comment:
Does this mean you support positions in Cartesian coordinates?

Reply:
If so, could you add a test for parsing such input file because I am not very familiar with qe. The test is `test/interface/test_pwscf.py`.

Reply:
Thanks for pointing this out, I fixed an error, added bohr units, and added the tests for the 2 new cases